samples.bluemix/TestRead/TestRead.spl

```diff
@@ -23,10 +25,10 @@ param file: "lines.txt";
     stream<PersonType> AsTuples = WebHdfsRead() {
     param
         url:getSubmissionTimeValue("url");
-        file: getSubmissionTimeValue("personfile","/tmp/TxtFile.txt");
+        file: getSubmissionTimeValue("personfile","/tmp/People.txt");
 	password: getSubmissionTimeValue("password");
 	username: getSubmissionTimeValue("username");
-	format: Parse.txt;
+	format: Parse.csv;
```

Comment:
What's the logic behind changing this?  `txt` is SPL's tuple format and for this file `csv` and `txt` both work.